### DISCUSSION TITLE
Zigbee2MQTT LXC

### DISCUSSION THREAD

**Discussion by bam27beer** [tteck/Proxmox#1383] [Help Needed]
must be going mad ,as i have this setup in home assistant
but by entering  
ls -l /dev/serial/by-id
either in the lxc shell ,or proxmox shell ,
i get no such file exists ? 
root@proxmox:~# ls -l /dev/serial/by-id
ls: cannot access '/dev/serial/by-id': No such file or directory
do i create a directory or copy from my existing hoas install ?
cheers in advance

**Comment by Tokn59** [comment]
Same here ... device (Sonoff Zigbee Dongle) shows up on pve level with lsusb ok but not in lxc ...

**Comment by tteck** [comment]
https://github.com/tteck/Proxmox/discussions/1388
